### PR TITLE
Add CLI tests for 2 parameters in factory.make_compute_resource - issue #1715

### LINKED ISSUE #1715: Add CLI tests for two new parameters in factory.make_compute_resource
The two new parameters are:
```
--location-ids LOCATION_IDS   REPLACE locations with given ids
--set-console-password        SET_CONSOLE_PASSWORD for Libvirt and Vmware only
                                                   One of true/false,
                                                   yes/no, 1/0.
```

### PR DESCRIPTION
**Pull request by abalakh**
Added CLI tests for two new parameters in factory.make_compute_resource:
```
--location-ids LOCATION_IDS   REPLACE locations with given ids
--set-console-password        SET_CONSOLE_PASSWORD for Libvirt and Vmware only
                                                   One of true/false,
                                                   yes/no, 1/0.
```
Resolves #1715

### CODE REVIEW

File: tests/foreman/cli/test_computeresource.py [hunk 1]
```diff
@@ -159,6 +163,52 @@ def test_create_positive_libvirt(self, options):
...
+                'location-ids': location['id'],
+            })
+        except CLIFactoryError as err:
+            self.fail(err)
+        self.assertEqual(1, len(comp_resource['locations']))
+        self.assertEqual(comp_resource['locations'][0], location['name'])
+
+    @skip_if_bug_open('bugzilla', 1214312)
+    @data(u'True', u'Yes', 1, u'False', u'No', 0)
+    def test_create_compute_resource_with_console_password(self, data):
```
Comment (Ichimonji10) [inline tests/foreman/cli/test_computeresource.py]: Please do not use a variable named `data`. You're redefining the `data` that is imported at the top of this module.

**Comment by elyezer** [comment]
ACK

**Comment by Ichimonji10** [comment]
ACK pending comment.

Also, it's really nice when test results are posted in the PR. It makes it easier to judge the quality of a PR.

```
$ nosetests tests/foreman/cli/test_computeresource.py -m test_create_compute_resource_with_location
.
----------------------------------------------------------------------
Ran 1 test in 7.898s

OK
```

**Comment by abalakh** [comment]
Updated according to comments. 
@Ichimonji10 i know, i do usually include test results. it's just this time 6 out of 7 tests are skipped due to a defect :)
Here're results for the 7th test:
```
$ nosetests tests/foreman/cli/test_computeresource.py -m test_create_compute_resource_with_location
.
----------------------------------------------------------------------
Ran 1 test in 13.141s

OK
```

**Comment by Ichimonji10** [comment]
Looks great. Thank you, @ablakh.

**Pull request merged**